src/model/PhoneNumber.ts

```diff
@@ -0,0 +1,5 @@
+export default interface PhoneNumber {
+  id: string
+  phoneNumber: string
+  type: string
```

Comment:
There are only a certain number of types allowed for a phone (and address and email). I think the type of this should be an enum, representing the different types. 

We should then use that Enum to populate the dropdowns. That way, our models and dropdowns require a change in one spot to stay in-sync.